modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -393,7 +347,40 @@ The previous error occurs because MIME types are not preserved by default, and t
 
 By setting this attribute, Mule automatically maintains each record's media type and knows that the payload actually represents a JSON document.
 
-== Max Concurrency Limit
+== Changing the Record Block Size
```

Comment:
This section was moved, but only the XML example was edited/corrected. Likely needs additional editing work in the future. 

Moved because it was talking about block sizes for the Batch Job scope, _not_ the Size attribute of the Batch Aggregator. However, it did mistakenly have an XML example with a size setting on the aggregator. That was replaced because it didn't belong.